releases.yml

```diff
@@ -24,6 +24,14 @@
       url: /docs/patterns/tiered-list
       status: Updated
       notes: We've added new CTA block usage examples for the tiered list pattern.
+    - component: Tiered list
+      url: /docs/patterns/tiered-list
+      status: Updated
+      notes: The tiered list pattern macro has been published.
+    - component: Hero
+      url: /docs/patterns/hero
+      status: Updated
+      notes: The hero pattern macro has been published.
```

Comment:
Adding these here doesn't help much until the these macros are documented ([WD-14523](https://warthogs.atlassian.net/browse/WD-14523) and [WD-14522](https://warthogs.atlassian.net/browse/WD-14522)).

Maybe the target branch of this PR should be set to the feature branch `feature/expose-macros` so we can make sure we release them all at once?